witness/golang/omniwitness/usbarmory/usbnet.go

```diff
@@ -0,0 +1,133 @@
+// https://github.com/f-secure-foundry/tamago-example
+//
+// Copyright (c) F-Secure Corporation
+// https://foundry.f-secure.com
+//
+// Use of this source code is governed by the license
+// that can be found in the LICENSE file.
+
+//go:build usbarmory
+// +build usbarmory
+
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+
+	usbnet "github.com/f-secure-foundry/imx-usbnet"
+	"github.com/f-secure-foundry/tamago/soc/imx6/usb"
+	"github.com/golang/glog"
+	"github.com/miekg/dns"
+)
+
+const (
+	deviceIP  = "10.0.0.1"
+	deviceMAC = "1a:55:89:a2:69:41"
+	hostMAC   = "1a:55:89:a2:69:42"
+	resolver  = "8.8.8.8:53"
+)
+
+var iface *usbnet.Interface
+
+func initNetworking() (net.Listener, error) {
+	var err error
+
+	iface, err = usbnet.Init(deviceIP, deviceMAC, hostMAC, 1)
+
+	if err != nil {
+		return nil, fmt.Errorf("could not initialize USB networking, %v", err)
+	}
+
+	iface.EnableICMP()
+
+	l, err := iface.ListenerTCP4(80)
+	if err != nil {
+		return nil, fmt.Errorf("could not initialize HTTP listener, %v", err)
+	}
```

Comment:
Should we comment that out / ding it?

Reply:
Ok, maybe I need to look at this code again - this appears to be opening a random listen on a hard-coded port 80, but the witness is listening on 8080 so I assumed this was unused cruft that came in with the usbnet code.

Reply:
Ah, that `8080` was the `monolith` main, my bad.